internal/controller/discoverymgr/mnedc/server/mocks/mocks_server.go

```diff
@@ -1,229 +1,226 @@
-/*******************************************************************************
- * Copyright 2020 Samsung Electronics All Rights Reserved.
```

Comment:
@sun-sharma Any reason to omit the copyright statement?

Reply:
No, mocks were auto-generated and thus it would have got removed. I will be regenerating them for the next commit on this PR. I will add this back.